src/OpenTelemetry.Extensions.Docker/Resources/DockerSemanticConventions.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="DockerSemanticConventions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Extensions.Docker.Resources
+{
+    internal static class DockerSemanticConventions
+    {
+        public const string AttributeContainerID = "container.id";
```

Comment:
please add a link to the relevant OTel spec/convention

Reply:
I have added readme file which provides the usage/description as a part of previous PR - https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Extensions.Docker

I'm not sure if you mean by updating the OpenTelemtry docs - https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/extending-the-sdk/README.md. If I need to update it here, Do I need to provide an example as well?